README.md

```diff
@@ -72,6 +72,7 @@ For additional help, join us [in our Slack](https://now-examples-slackin-rrirkqo
 -   [React](app/react)
 -   [React Native](app/react-native)
 -   [Vue](app/vue)
+-   [Svelte](app/svelte)
```

Comment:
Please place it after the HTML and add the `alpha` tag (for the chronological order)